usaspending_api/templates/plate.html

```diff
@@ -0,0 +1,124 @@
+{% load docs %}
```

Comment:
Thanks for incorporating this so nicely into the overall docs!

Eventually we should see if we can figure out how to make up the hover-over info on fields more legible, but that's very non-urgent.